### ISSUE TITLE
Screen reader cannot close tabs in the script window

### ISSUE THREAD

**Issue by hturner** [r-wasm/webr#274]
Comment from @ajrgodfrey

> A screen reader cannot close tabs in the script window: the cross icon is not detectable or selectable.

**Comment by georgestagg** [comment]
As of 7df7964 the cross icon is contained in a real `<button>` element. In my testing, I am able to navigate to the cross using Tab navigation.

(It will take a few hours for the webR website to update, but the changes should be available by tomorrow.)
